commcare_connect/templates/reports/htmx_table.html

```diff
@@ -0,0 +1,87 @@
+{% extends "django_tables2/bootstrap5.html" %}
```

Comment:
is it possible to reuse (or inherit) from the base report table we already defined?

Reply:
This template is just the table part, the base report table template includes the filters and other nav-sections etc..